content/components/privately-shared-components.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Privately shared components
```

Comment:
I think we should discuss this with @emilybrick and run this through a "what misunderstanding that can occur" if we publish this on our public page and make sure this is in alignment with the rest of our docs strategy. Also, "Private" and "Shared" are opposites, perhaps we can find a name that provides more clarity.

Reply:
I'm good with `GitHub shared components` fwiw.